recipes/imapclient/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{%set name = "IMAPClient" %}
+{%set version = "1.0.2" %}
+{%set compress_type = "tar.gz" %}
```

Comment:
Hmm. ok. Honestly, `compress_type` is really in there because it'd be nice to have a single variable that covers name-version.compression (sometimes `zip`)  and I couldn't get a variable defined in terms of other, extent variables.